packages/pwa/app/components/footer/index.jsx

```diff
@@ -188,7 +188,7 @@ const Subscribe = ({...otherProps}) => {
             <Heading {...styles.subscribeHeading}>
                 {intl.formatMessage({
                     id: 'footer.subscribe.heading.first_to_know',
-                    defaultMessage: 'Be the first to know'
+                    defaultMessage: 'Be the First to Know'
```

Comment:
This looks like a sentence to me. "Be the first to know" looks better

Reply:
Cornelia mentioned the line "You Might Also Like" for this translation, which to me should not be in title case since it is more like a sentence, I don't see any reason why we should title case. It is better to change "You Might Also Like" to "You might also like" than change the "Be the first to know" line